### ISSUE TITLE
UInt64 duplicates functionality in guava

### ISSUE THREAD

**Issue by lookfirst** [PegaSysEng/artemis#34]
https://github.com/PegaSysEng/artemis/blob/master/util/src/main/java/tech/pegasys/artemis/util/uint/UInt64.java

https://github.com/google/guava/blob/master/guava/src/com/google/common/primitives/UnsignedLong.java

**Comment by jrhea** [comment]
Yep. Totally correct. We were discussing the same thing. Thanks for the feedback

**Comment by schroedingerscode** [comment]
While implementing our own 64-bit unsigned int was an interesting exercise, given that we are already using Guava for our events, I see no reason to not use the Guava implementation.

**Comment by cleishm** [comment]
Never quite understood why Guava used different method names from `BigInteger` (e.g. `plus` vs `add`, `minus` vs `subtract`). FWIW, https://github.com/ConsenSys/cava/blob/master/units/src/main/java/net/consensys/cava/units/bigints/UInt256.java uses the `BigInteger` naming. But I'd agree that keeping it simple and reusing existing things is the best place to start!
